### DISCUSSION TITLE
Unauthorized error when trying to push branch to the repository

### DISCUSSION THREAD

**Discussion by pragyamutluru** [micronaut-projects/micronaut-core#10997] [Q&A] [answered]
Hi,

It is my first time contributing to the repository.
However, I am getting 403 Unauthorised when I try to push a branch related to the issue.

Is there any particular access I need to raise or a process to follow?
There is no special access mentioned in https://github.com/micronaut-projects/micronaut-core/blob/4.6.x/CONTRIBUTING.md

**Comment by samarth-ramesh** [comment, marked as answer]
Are you trying to push to your own fork? 
Or are you trying to push to micronauth-projects/micronaut-core?

**Reply by pragyamutluru** [reply]
I tried to push to micronaut-projects/micronaut-core.
I think it makes sense for me to fork and then raise the Pull Request.
Did that now and that worked!

Thank you!
